src/components/Export.tsx

```diff
@@ -219,12 +219,11 @@ export default function Export(inputProps: ExportButtonProps): ReactElement {
       cancelText: "Back",
       centered: true,
       icon: null,
-      getContainer: modalContainerRef || undefined,
       onOk: () => {
```

Comment:
The drop-in replacement for the modal API worked so well that GitHub isn't showing the invocation lol. The new modal wrapper is being called on line `215` if you're curious.